src/picongpu/main.cpp

```diff
@@ -63,6 +64,8 @@ MALLOCMC_SET_ALLOCATOR_TYPE( ScatterAllocator );
 
 #else
 
+#define PMACC_ACC_CPU
```

Comment:
It is deleted in the comment below. I already had added the (hopefully) correct version.